spaceMissionManager-persistance/src/main/java/cz/muni/fi/Entity/CraftComponent.java

```diff
@@ -1,12 +1,75 @@
 package cz.muni.fi.Entity;
 
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
 import java.time.ZonedDateTime;
 
+@Entity
 public class CraftComponent {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
 
+	@Column
+	@NotNull
 	private boolean readyToUse;
+
+	@Column
+	@NotNull
 	private String name;
+
 	private ZonedDateTime readyDate;
-	private Long id;
 
+	public Long getId() {
+		return id;
+	}
+
+	public void setId(Long id) {
+		this.id = id;
+	}
+
+	public boolean isReadyToUse() {
+		return readyToUse;
+	}
+
+	public void setReadyToUse(boolean readyToUse) {
+		this.readyToUse = readyToUse;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public ZonedDateTime getReadyDate() {
+		return readyDate;
+	}
+
+	public void setReadyDate(ZonedDateTime readyDate) {
+		this.readyDate = readyDate;
+	}
+
+
+	@Override
+	public boolean equals(Object o) {
+		if (this == o) return true;
+		if (o == null || getClass() != o.getClass()) return false;
+
+		CraftComponent that = (CraftComponent) o;
+
+		if (isReadyToUse() != that.isReadyToUse()) return false;
```

Comment:
I think that "primary key" (or the set that defines equality) does not need to include this bool, what do others think?
What defines unique component?